tests/test_parsl_flux_mpi_hello.py

```diff
@@ -123,6 +123,12 @@ def test_run_mpi_hello(load_config):
                           ).result()
     assert hello == 0
 
+    with open('parsl_flux_mpi_hello_run.out', 'r') as f:
+        print("file", f.read())
```

Comment:
Currently the tests pass for me locally when adding this print statement in, but curiously fail without it. I wonder if this is because I am reading the input file into memory. Debugging this issue and feel like it should be a simple fix to address.

Reply:
When you are iterating line by line, you probably don't want to use `f.read()`.  The `for line in f` takes care of reading each line one by one.  So, in your assert, I think you'll want to compare to `line` instead of `f.read()`.

Reply:
Updated to `line` instead of `f.read()`